rcl_yaml_param_parser/include/rcl_yaml_param_parser/parser.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCL_YAML_PARAM_PARSER__PARSER_H_
+#define RCL_YAML_PARAM_PARSER__PARSER_H_
+
+#include <stdlib.h>
+
+#include "rcl_yaml_param_parser/types.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/// \brief Parse the YAML file and populate params_hdl
```

Comment:
What is `params_hdl` here?

Reply:
Ahh... Initially I had named the parameter as params_hdl, where it is a handle. That makes more sense when the structure is hidden and the API just retrurns a void pointer. In this case since the structure is in the header, it does not make much sense. Anyways, have fixed it